modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -1463,4 +1475,7 @@ const MySkeleton = () => (
 
 Additionally, the skeleton animation has been updated from a diagonal sheen, or shimmer, to fading
 the opacity of the entire shape(s) in and out.
+
+```
```

Comment:
This keeps happening. Not sure what's up.

Reply:
I think a formatter is closing the doc block